core-runner/src/runner.rs

```diff
@@ -1256,7 +1283,7 @@ mod tests {
 
         assert_eq!(
             result.gas_spent[0].1,
-            runner.alloc_cost() * 1 + runner.load_page_cost() * 1 + 3000
+            runner.alloc_cost() * 2 + runner.load_page_cost() * 1 + 3000
```

Comment:
If I understand changes correctly, shouldn't it be `runner.alloc_cost() + runner.mem_grow_cost()`? Test may fail if default values are changed.

Reply:
A yep. Forget about this test